consumer.ts

```diff
@@ -54,6 +54,14 @@ const sendEachMessage = async ({ message }) => {
   return;
 };
 
+// Instanciate `Consumer` function every 45 seconds to avoid running before producer & Kafka does
+const consumerThrottle = async () => {
+  setTimeout(() => {
+    console.log("Kafka not ready, consumer waiting to load");
```

Comment:
Line 60 seems like its in the wrong place, this will only fire after the timeout, so the message doesnt make sense